### PR TITLE
Add huawei to adopters

### PR DESCRIPTION
**Pull request by Poor12**
Signed-off-by: Poor12 <shentiecheng@huawei.com>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Add huawei to adopters

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
None

### CODE REVIEW

File: docs/casestudies/adopters.md [hunk 1]
```diff
@@ -12,13 +12,14 @@ To join this list, please follow [these instructions](https://github.com/karmada
...
+|---------------------------|------------------------------|-----------------------------------------------------------------------------------------|-------------------------------------------------------------------------------|
+| Cibfintech.com            | https://www.cibfintech.com/  | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada            | TBD                                                                           |
+| DaoCloud.io               | https://www.daocloud.io/     | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada            | TBD                                                                           |
+| Ecloud                    | https://ecloud.10086.cn/     | Use Karmada to build a steady and elastic multi-cluster platform for managing workloads | TBD                                                                           |
+| Huawei                    | https://www.huaweicloud.com/ | Build an enterprise-grade multi-cloud container platform with Karmada                   | TBD                                                                           |
```
Comment (RainbowMango) [inline docs/casestudies/adopters.md:20]: @GitHubxsy Any comments? 
I think a link to the service would be great.

Comment (GitHubxsy) [reply docs/casestudies/adopters.md:20]: [Ubiquitous Cloud Native Service（UCS）](https://www.huaweicloud.com/product/ucs.html)   
UCS product link：https://www.huaweicloud.com/product/ucs.html

Comment (Poor12) [reply docs/casestudies/adopters.md]: Done

**Review by Arhell — COMMENTED** [review]
/lgtm

File: docs/casestudies/adopters.md [hunk 2]
```diff
@@ -12,13 +12,14 @@ To join this list, please follow [these instructions](https://github.com/karmada
...
+| DaoCloud.io               | https://www.daocloud.io/     | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada                                                                                   | TBD                                                                           |
+| Ecloud                    | https://ecloud.10086.cn/     | Use Karmada to build a steady and elastic multi-cluster platform for managing workloads                                                                        | TBD                                                                           |
+| Huawei                    | https://www.huaweicloud.com/ | Build an enterprise-grade distributed cloud native platform([Ubiquitous Cloud Native Service(UCS)](https://www.huaweicloud.com/product/ucs.html)) with Karmada | TBD                                                                           |
```
Comment (RainbowMango) [inline docs/casestudies/adopters.md:20]: Please talk to @GitHubxsy about the usage scenario. 
/assign @GitHubxsy

File: docs/casestudies/adopters.md [hunk 3]
```diff
@@ -11,15 +11,15 @@ To join this list, please follow [these instructions](https://github.com/karmada
...
+| DaoCloud.io               | https://www.daocloud.io/      | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada                                                                                   | TBD                                                                           |
+| Ecloud                    | https://ecloud.10086.cn/      | Use Karmada to build a steady and elastic multi-cluster platform for managing workloads                                                                        | TBD                                                                           |
+| Huawei                    | https://www.huaweicloud.com/  | Build an enterprise-grade distributed cloud native platform([Ubiquitous Cloud Native Service(UCS)](https://www.huaweicloud.com/product/ucs.html)) with Karmada | TBD                                                                           |
```
Comment (RainbowMango) [inline docs/casestudies/adopters.md:19]:
```suggestion
| Huawei                    | https://www.huaweicloud.com/  | Built the [Ubiquitous Cloud Native Service](https://support.huaweicloud.com/intl/en-us/productdesc-ucs/ucs_productdesc_0001.html) that run applications across regions and clouds with intelligent traffic distribution. | TBD                                                                           |
```

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**